bin/fmt.sh

```diff
@@ -24,6 +24,6 @@ done
 gofmt -s -w ${GO_FILES}
 goimports -w -local istio.io ${GO_FILES}
 buildifier -mode=fix $(find . -name BUILD -type f)
-buildifier -mode=fix ./BUILD.api
+buildifier -mode=fix ./*.bzl
```

Comment:
BUILD.api file was removed in the last commit and replaced with severl .bzl files.
This change ensures that fmt.sh runs to completion.